### PR TITLE
Make IPC Sounds Great Again

### PR DESCRIPTION
**Pull request by BasedUser**
# Description

This PR brings back Ekrixi IPC sounds, and rebases IPC sounds on top of silicons.  
Duplicate of #810 [due to](# "Due to your attempt to evade your ban, you are banned indefinitely. You may appeal your ban, but only at least 6 months after your last ban evasion attempt, and only with a voucher of good behavior from another SS13/SS14 server.") `master` branch skill issues.

<details open><summary><h2>Media</h2></summary>
None
</details>

---

# Changelog

:cl: router
- add: Added more sounds to IPCs. IPCs can now also whistle, whirr, beep, boop, ping, chime, buzz and buzz twice.
- fix: IPCs no longer have the default wilhelm sound.
- tweak: Silicon deathgasps are now darker.

### CODE REVIEW

File: Resources/Audio/Voice/IPC/wilhelm.ogg
Comment (FoxxoTrystan) [inline Resources/Audio/Voice/IPC/wilhelm.ogg:1]: This file is not attributed in Attributions.yml, is this normal?

Comment (FoxxoTrystan) [reply Resources/Audio/Voice/IPC/wilhelm.ogg:1]: And this is not gona be annoying to hear ingame sometimes.

**Review by VMSolidus — APPROVED** [review]
This is fine, test fails can be ignored.

**Review by FoxxoTrystan — APPROVED** [review]

**Review by OldDanceJacket — APPROVED** [review]

File: Resources/Audio/Voice/IPC/wilhelm.ogg
Comment (BasedUser) [reply Resources/Audio/Voice/IPC/wilhelm.ogg:1]: It's literally just beep1.ogg several times. There's no attribution for beep1, so I'm not attributing this either.

Yes, the PR was already merged, but I hate seeing an unresolved review.

**Pull request merged**